bikeshed/ReferenceManager.py

```diff
@@ -257,14 +257,14 @@ def getRef(self, linkType, text, spec=None, status=None, linkFor=None, error=Tru
             # Allow foo(bar) to be linked to with just foo(), but only if it's completely unambiguous.
             textPrefix = text[:-1]
             candidates, _ = self.queryRefs(linkType=linkType, status="local", linkFor=linkFor)
-            urls = list(set(c.url for c in candidates if c.text.startswith(textPrefix)))
-            if len(urls) == 1:
-                return urls[0]
+            candidateRefs = list(set(c for c in candidates if c.text.startswith(textPrefix)))
```

Comment:
This needs to preserve the url-uniquifying behavior.  There are several instances where different refs have the same url, so warning about multiple refs is just an annoyance.  This can be done by switching to a url-keyed dict that only preserves one of the references, then calling `list(refs.values())`.